pkg/apply/event/event.go

```diff
@@ -85,6 +89,10 @@ type ErrorEvent struct {
 	Err error
 }
 
+type WaitEvent struct {
+	Error error
```

Comment:
If we decide to keep this new event, please add a GroupName to match other events and can be associated with a parent ActionGroup

Reply:
Not all the event types have GroupName.
What is the purpose of GroupName?

Reply:
Group name is used to correlate apply/prune/delete/wait events (usually for individual objects) with action groups (tasks on groups of objects). It's required to differentiate when there are multiple task of the same type and helps sorting for testing comparison.

Reply:
Thanks for the explanation. Added it.